bigtable-hbase/src/main/java/org/apache/hadoop/hbase/client/AbstractBigtableConnection.java

```diff
@@ -68,6 +72,39 @@
       Collections.synchronizedMap(new HashMap<Long, BigtableBufferedMutator>());
 
   static {
+    // Initialize some core dependencies in parallel.  This can speed up startup by 150+ ms.
+    ExecutorService connectionStartupExecutor =
+        Executors.newCachedThreadPool(
+            new ThreadFactoryBuilder()
+                .setNameFormat("AbstractBigtableConnection-startup-%s")
+                .setDaemon(true)
+                .build());
+    connectionStartupExecutor.execute(new Runnable() {
+      @Override
+      public void run() {
+        // Startup of SSL_CONTEXT_FACTORY is a really expensive operation for the first operation.
+        // It could take up to 2 seconds.
+        BigtableOptions.SSL_CONTEXT_FACTORY.create();
```

Comment:
This returns an SslContext, maybe do a similar thing like the other run() so that it is obvious we are createing an object and then discarding it. Also, is it ok to create SslContext and not use them?